locale/es_MX/LC_MESSAGES/clr-installer.po

```diff
@@ -474,10 +474,10 @@ msgid "Passphrases do not match"
 msgstr "Las frases de contraseña no coinciden"
 
 msgid "Mapping %s partition to an encrypted partition"
-msgstr "Asignación de la partición %s a una partición cifrada"
+msgstr "Asignación de la partición% s a una partición cifrada"
```

Comment:
I don't think this is valid -- they broke the `%s`

Reply:
Good catch. Lets check with them.